### PR TITLE
Add missing ring interface function for AlgClosure

### PR DESCRIPTION
**Pull request by thofma**

### CODE REVIEW

File: src/Rings/AlgClosureFp.jl [hunk 1]
```diff
@@ -44,6 +44,8 @@ end
 elem_type(::Type{AlgClosure{T}}) where T = AlgClosureElem{T}
 parent_type(::Type{AlgClosureElem{T}}) where T = AlgClosure{T}
 
+Oscar.canonical_unit(a::AlgClosureElem) = a
```
Comment (lgoettgens) [inline src/Rings/AlgClosureFp.jl]:
```suggestion
Oscar.canonical_unit(a::AlgClosureElem) = is_zero(a) ? one(a) : a
```
In fields with an easy iszero check we should use that to make sure to return a unit.

Comment (thofma) [reply src/Rings/AlgClosureFp.jl]: I just went with
```
julia> canonical_unit(QQ(0))
0
```

Comment (lgoettgens) [reply src/Rings/AlgClosureFp.jl]: There are multiple issues in AA/Nemo about that.
But returning the "unit" 0 can lead to division by zero easily (in not so well-written implementations)

**Review by lgoettgens — APPROVED** [review]

File: src/Rings/AlgClosureFp.jl (hunk 1 above)
Comment (thofma) [reply src/Rings/AlgClosureFp.jl]: Yes. It should be renamed and redesigned properly.

**Pull request merged**